azurerm/helpers/suppress/xml.go

```diff
@@ -0,0 +1,47 @@
+package suppress
+
+import (
+	"encoding/xml"
+	"io"
+	"reflect"
+	"strings"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func XmlDiff(_, old, new string, _ *schema.ResourceData) bool {
+	oldTokens, err := expandXmlTokensFromString(old)
+	if err != nil {
+		return false
```

Comment:
Just confirming that we don't want to handle about the errors that `expandXmlTokensFromString` returns

Reply:
since this is a diff suppress func it should be fine, it'll just mean a diff is shown vs no diff (which is fine) 👍